bot/main.py

```diff
@@ -271,11 +285,12 @@ async def roll(
     """
     user = db.users.find_one({"rollno": str(identifier)})
     if user:
-        await ctx.reply(
-            f"Name: {user['name']}\nEmail: {user['email']}\nRoll Number: {user['rollno']}"
+        await ctx.send(
+            f"Name: {user['name']}\nEmail: {user['email']}\nRoll Number: {user['rollno']}",
+            ephemeral=True,
         )
     else:
-        await ctx.reply(f"{identifier} is not registered with IIIT-CAS.")
+        await ctx.send(f"{identifier} is not registered with IIIT-CAS.", ephemeral=True)
```

Comment:
Any specific reason why the `reply` calls are being changed to `send` across this file?

Reply:
In the documentation of [ctx.send](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html?highlight=ephemeral#discord.ext.commands.Context.send), is the only one that mentions having ephemeral as a function. So I thought only ctx.send() would have it.

However today on testing with ctx.reply(), we can use ephemeral attribute there too, so I made a commit with the changes.